PULL_REQUEST_TEMPLATE.md

```diff
@@ -13,70 +13,77 @@
   Why are you making this change? There is nothing more important
   to provide to the reviewer and to future readers than the cause
   that gave rise to this pull request. Be careful to avoid circular
-  statements like "the purpose is to update the schema." and
-  instead provide an explanation like "there is more data to be provided and stored for Purchase Orders 
+  statements like "the purpose is to update the schema." 
+  instead, provide an explanation like "there is more data to be provided and stored for Purchase Orders 
```

Comment:
Why do we need a comma sign after "instead"?

Reply:
recommended by grammarly
instead provide an explanation like "there is more data to be provided and stored for Purchase Orders 
![image](https://user-images.githubusercontent.com/113523904/208088188-49b03dda-61c5-4140-aeae-b6c3055db6bf.png)